bfasst/yaml_parser.py

```diff
@@ -105,3 +108,52 @@ def __init__(self, yaml_path):
         self.vhdl_libs = None
         if "vhdl_libs" in self.props:
             self.vhdl_libs = self.props["vhdl_libs"]
+
+
+class FlowDescriptionParser(YamlParser):
```

Comment:
In my opinion it would be better to just perform the parsing in __init__ once and save the data into class properties rather than having the function do the parsing each time.  Thoughts?

Reply:
In this file, that is what I'm doing. I guess I don't understand what you mean?